dpdata/abacus/md.py

```diff
@@ -42,6 +41,36 @@ def get_coord_dump_freq(inlines):
 
 
 def get_coords_from_dump(dumplines, natoms):
+    """abacus version >=3.1.4:
+    MDSTEP:  0
+    LATTICE_CONSTANT: 12.411200939060 Angstrom
+    LATTICE_VECTORS
+      1.000000000000  0.000000000000  0.000000000000
+      0.000000000000  1.000000000000  0.000000000000
+      0.000000000000  0.000000000000  1.000000000000
+    VIRIAL (kbar)
+      36.689617311101  0.000000000003  -0.000000000000
+      0.000000000003  36.689617311089  -0.000000000001
+      -0.000000000000  -0.000000000001  36.689617311135
+    INDEX    LABEL    POSITION (Angstrom)    FORCE (eV/Angstrom)    VELOCITY (Angstrom/fs)
+      0  Sn  0.000000000000  0.000000000000  0.000000000000  -0.000000000000  -0.000000000001  -0.000000000001  0.001244557166  -0.000346684288  0.000768457739
+      1  Sn  0.000000000000  3.102800034079  3.102800034079  -0.000186795145  -0.000453823768  -0.000453823768  0.000550996187  -0.000886442775  0.001579501983.
+
+    abacus version < 3.1.4:
+    MDSTEP:  0
+    LATTICE_CONSTANT: 23.453780000000
+    LATTICE_VECTORS
+      1.000000000000  0.000000000000  0.000000000000
+      0.000000000000  1.000000000000  0.000000000000
+      0.000000000000  0.000000000000  1.000000000000
+    VIRIAL (KBAR)
+      36.689617311101  0.000000000003  -0.000000000000
+      0.000000000003  36.689617311089  -0.000000000001
+      -0.000000000000  -0.000000000001  36.689617311135
+    INDEX    LABEL    POSITIONS    FORCE (eV/Angstrom)
+      0  Sn  0.000000000000  0.000000000000  0.000000000000  -0.000000000000  -0.000000000001  -0.000000000001
+      1  Sn  0.000000000000  0.250000000000  0.250000000000  -0.000186795145  -0.000453823768  -0.000453823768
+    """
```

Comment:
We follow [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) to use docstring as documentation. Please do not leave raw data in the docstring.

Reply:
Ok, it is removed.